server/client.go

```diff
@@ -286,9 +286,10 @@ type rrTracking struct {
 
 // Struct for PING initiation from the server.
 type pinfo struct {
-	tmr  *time.Timer
-	last time.Time
-	out  int
+	tmr        *time.Timer
+	lastRxPing time.Time
```

Comment:
Can these not be combined? I think they are doing the same which is recording inbound activity only.